pym/bob/input.py

```diff
@@ -2463,6 +2489,9 @@ def __parse(self):
         self.__createSchemas()
 
         # user config(s)
+        self.__parseUserConfig("/etc/bobdefault.yaml")
```

Comment:
Do you think it is useful to have includes for system / user default's? If so I think we should break backwards compatibility cause relative to current working directory intuitive. Otherwise it may be easier to make them only available for the workspace default.yaml.